modules/ossm-configuring-external-jaeger.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+
+[id="ossm-specifying-external-jaeger_{context}"]
+= Specifying Jaeger configuration in a Jaeger custom resource
+
+Configuring Jaeger in the Jaeger customer resource rather than in the `ServiceMeshControlPlane` lets you fully customize your Jaeger deployment.  This configuration is sometimes referred to as an "external Jaeger" since the configuration is specified outside of the SMCP.
```

Comment:
"customer" should be "custom"

Reply:
LOL, there's always at least one typo!  This is why I need reviewers.
Fixed.